vm/src/pyobject.rs

```diff
@@ -384,6 +384,17 @@ impl PyContext {
         PyObject::new(PyInt::new(i), self.int_type(), None)
     }
 
+    #[inline]
+    pub fn new_bigint(&self, i: &BigInt) -> PyObjectRef {
```

Comment:
@coolreader18 Do you have any idea to integrate this function into new_int?

Reply:
You could maybe declare your own trait and implement it for both `T where T: Into<BigInt> + ToPrimitive` and `&T where T: Into<BigInt> + ToPrimitive`. I think the real issue is that `ToPrimitive` should also be implemented by `num-traits` for `&T where T: ToPrimitive`.

Reply:
For now, I think I am not interested in working for trait puzzle.
Would you review this PR as is again please?